topics/transcriptomics/tutorials/scrna_preprocessing/tutorial.md

```diff
@@ -0,0 +1,1244 @@
+---
+layout: tutorial_hands_on
+topic_name: transcriptomics
+tutorial_name: scrna
+---
+
+<style>
+blockquote img {
+  width: 50%;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+.tutorial table { 
+  width:unset;
+  margin-left:auto;
+  margin-right:auto;
+}
+
+.tutorial table thead th {
+  border-bottom: 2px solid #777; 
+  text-align: left;
+}
+
+.tutorial table td:first-child, .tutorial table th:first-child  {
+  border-right: 2px solid #777; 
+  font-weight: bold;
+  text-align: left;
+}
+
+.tutorial table td:not(first-child), .tutorial table th:not(first-child) {
+  text-align: center;
+  border-right: 1px solid #ddd;
+  border-left: 1px solid #ddd;
+}
+
+</style>
+
+
+# Introduction
+{:.no_toc}
+
+The advent of single-cell RNA sequencing has provided the means to explore samples at the individual cell level, enabling a greater understanding of the development and function of such samples by the characteristics of their constituent cells. These cells can be classified into different cell subpopulations or "clusters" by the variation of gene expression exhibited by individual cells that share a common profile -- i.e. cells in the same cluster exhibit similar profiles of differential expression in the same set of related genes.
```

Comment:
Is a single cell really a single cell or just cells from one group or one tissue? If a single cell is defined by similar gene expression can they be "sorted" already before having the expression profile?